client/modules/wj-css/src/components/_links.scss

```diff
@@ -0,0 +1,49 @@
+@use "sass:list";
+@use "sass:selector";
+
+a[href]:not([href*="javascript:;"]) {
+  // Base link styling.
+  text-decoration: currentColor underline 0.1em;
+  transition: color 0.05s, text-decoration-color 0.1s;
+
+  // TODO: replace hardcoded links with some other system
+  // for now this will work, but it's a little messy
+  // prettier-ignore
+  $internal-links-list:
+    '^="/"',
+    '^=".."',
+    '^="#"',
+    '*="wikijump.com"',
+    '*="wikijump.org"',
+    '*="wikijump.test"',
+    '*="wikijump.dev"',
+    '*="localhost"';
+
+  // make our selector lists
+  $is-internal: ();
+  $not-internal: "&[href*='//']";
```

Comment:
Would `://` be more correct?

Also technically stuff like `mailto:` exists even though it doesn't have `//` in it.

Reply:
I think if you do `//` at the start of a URL it means "pick a protocol for me", which is why I excluded the `:`. Although, when I test that in Firefox both `://` and `//` work.

Also, totally forgot about stuff like `mailto:` lol